src/Compilers/Core/Portable/NativePdbWriter/ComMemoryStream.cs

```diff
@@ -32,9 +42,10 @@ public void CopyTo(Stream stream)
             }
 
             int chunkIndex = 0;
-            for (int remainingBytes = _length; remainingBytes > 0;)
+            int remainingBytes = _length;
+            while (remainingBytes > 0)
             {
-                int bytesToCopy = Math.Min(ChunkSize, remainingBytes);
+                int bytesToCopy = Math.Min(_chunkSize, remainingBytes);
```

Comment:
>_chunkSize [](start = 43, length = 10)

It feels like this should be an actual length of the array representing the chunk. Also, if we already run out of chunks, it doesn't look useful to limit bytesToCopy to _chunkSize.